Restyle PLIBMTTBHGATY homepage

conchan opened the pull request:
Large scroll
![scroll](https://cloud.githubusercontent.com/assets/662225/14194506/2bda2816-f77d-11e5-96ca-4732910db696.gif)

Large
<img width="1677" alt="screen shot 2016-03-31 at 8 11 33 pm" src="https://cloud.githubusercontent.com/assets/662225/14194462/ceec8b94-f77c-11e5-9bfb-35b33b6f50fe.png">

Medium
<img width="1049" alt="screen shot 2016-03-31 at 8 12 37 pm" src="https://cloud.githubusercontent.com/assets/662225/14194476/f2430122-f77c-11e5-90d5-61f36de231b1.png">

Small
![mobile](https://cloud.githubusercontent.com/assets/662225/14194489/0cc9bcc0-f77d-11e5-8f92-88772949c867.png)

conchan committed:
Redesign PLIBMTTBHGATY site

Let's make it more fun and friendly! This commit is the redesign MVP: contains the basic information and links, as well as visual redesign.

conchan commented:
To do list
- add past events
- add photos

malandrina commented on source/partials/_footer.erb:
```diff
@@ -1,3 +1,11 @@
+    <ul class="social-list">
+      <li class="social twitter"><a href="http://twitter.com/plibmttbhgaty">Twitter</a></li>
+      <li class="social github"><a href="http://github.com/plibmttbhgaty">Github</a></li>
+      <li class="social email"><a href="mailto:star+plibmttbhgaty@mit.edu">Email</a></li>
```
Typo: Email should be "stars+plibmttbhgaty@mit.edu"

malandrina commented on source/stylesheets/_footer.scss:
```diff
@@ -0,0 +1,8 @@
+  margin-top: $base-spacing;
+  text-align: right;
+}
+  
```
Style: Extra newline

malandrina commented on source/index.html.erb:
```diff
@@ -3,67 +3,61 @@ pageable: true
+      Signs with names of programming languages
+    </p></li>
+    <li class="party-ingredient"><p>
+      A show-and tell at the end
```
Typo: "show-and-tell"

malandrina commented:
This is fantastic! :apple: :banana: :pineapple:

You've done a great job condensing all the information previously in the "About", "Events", and "Get Involved" sections into one fun, readable page. I love the choices you made for the colors, the background, the fonts, and the animations. The overall vibe is friendly and kind of wacky, which I like. Adding photos will give it a human presence.

Some questions:
- I like the friendliness of the Caveat font but I find it a little difficult to read. I'm curious what others think?
- Could we add information about who's hosting the events in the Events section? I think adding the host name will help people know if the event is going to be in a location that's of interest to/convenient for them.

![screen shot 2016-04-01 at 8 54 32 am](https://cloud.githubusercontent.com/assets/1855523/14212636/b752ed90-f7e8-11e5-88a3-d640c520e736.png)
- What's a good place for the code of conduct? I could see linking to it near the social links. I'm not sure what we're doing for a code of conduct yet -- the simplest thing would be to use the [contributor covenant](http://contributor-covenant.org/version/1/4/code_of_conduct.md).

malandrina commented on source/partials/_footer.erb:
```diff
@@ -1,3 +1,11 @@
+    </ul>
+  </nav>
+
   <p>&copy; <%= Date.today.year %> PLIBMTTBHGATY</p>
```
Maybe add a "designed [with love] by thoughtbot" or "designed by Connie Chan" because you've been spending investment time on this? If we don't want to add that here, maybe we can mention this design work on the "Products & Open Source" section of the new thoughtbot website.

conchan commented:
@malandrina thanks for the feedback! I changed up the typeface and added a link to thoughtbot. good idea about putting it on our site as well. 

I figure the code of conduct could be on this page too, where you had it on the bottom before. But it just said "TBD" so I removed it until we have one to put :) 

I agree the events listing could be more informative and interesting. Does the host name = location?

New footer:
<img width="467" alt="screen shot 2016-04-03 at 10 39 26 pm" src="https://cloud.githubusercontent.com/assets/662225/14237077/f736f3ec-f9ec-11e5-9880-3668f6923bc7.png">

New copy typeface:
<img width="1309" alt="screen shot 2016-04-03 at 10 39 32 pm" src="https://cloud.githubusercontent.com/assets/662225/14237079/fa737e0e-f9ec-11e5-9f46-a744d7a08664.png">

conchan commented:
Trello card: https://trello.com/c/ie0IqHpU/6-general-site-redesign

malandrina commented:
@conchan looks great!

> I figure the code of conduct could be on this page too, where you had it on the bottom before. But it just said "TBD" so I removed it until we have one to put :)

That sounds good. :+1: 

> I agree the events listing could be more informative and interesting. Does the host name = location?

Yes. Seems like the city, date, and name of the host (e.g. thoughtbot) is probably enough to give users an idea of whether attending is feasible for them, then they can click through to the RSVP link to view specific location details. Does that make sense?

malandrina commented:
@conchan it seems like the only outstanding item is to add the host names to the list of events; feel free to do that in another PR if you want!
